### PR TITLE
Hello button

### PR DESCRIPTION
**Pull request by osmolka**
Added a button that sends a request to the backend

### CODE REVIEW

**Comment by IharYudziankou** [comment]
I believe .idea folder was added by mistake. fixed it

File: src/components/App.js [hunk 1]
```diff
@@ -1,12 +1,27 @@
...
 class App extends Component {
+    constructor(props) {
+        super(props);
+
+        this.sendRequest = this.sendRequest.bind(this);
+    }
+
+    sendRequest() {
+        //TODO Replace string with env var process.env.REACT_APP_API_URI
+        get("http://localhost:8080" + '/init', (response) => {
```
Comment (IharYudziankou) [inline src/components/App.js]: can we get "http://localhost:8080" from properties?

Comment (osmolka) [reply src/components/App.js]: Yes, it will be from environment variables files (process.env.REACT_APP_API_URI)

**Comment by osmolka** [comment]
> I believe .idea folder was added by mistake. fixed it
It's a mistake, i've already fixed it.

**Comment by osmolka** [comment]
Hi. See my last commit, I added localhost's URI as an environment variable.
@IharYudziankou I used "dotenv-webpack" plugin that provides usage .env variables in code. You should only install "dotenv-webpack" (add it to package.json) and create .enf file.

**Pull request merged**